tests/features/movement_test.py

```diff
@@ -440,3 +440,46 @@ def test_indentation_using_tabs(run, tmpdir):
         h.await_cursor_position(x=4, y=2)
         h.press('Up')
         h.await_cursor_position(x=4, y=1)
+
+
+def test_goto_next_paragraph(run, tmpdir):
+    f = tmpdir.join('f')
+    f.write(
+        'class Test:\n\tpass\n\n\ndef another_test():'
+        '\n\tpass\n\nyet_another_test = None\n\n# comment',
+    )
+
+    with run(str(f)) as h, and_exit(h):
+        # Alt + Down Tests
+        h.press('\x1b[1;3B')
```

Comment:
these should use the "names" of these key presses -- I believe this would be `M-Up` ?

the fake runner will need adjustment to support those, it has a mapping of key bindings

Reply:
Tried to replace with `M-Down` and `M-Up`, but then the tests fail because of `unknown key: Key(wch='\x1bDown', keyname=b'unknown')`

Reply:
yep! you'll need to add support for the new key in `tests/features/conftest.py`